doc/source/basics.rst

```diff
@@ -1755,12 +1755,14 @@ then the more *general* one will be used as the result of the operation.
    # conversion of dtypes
    df3.astype('float32').dtypes
 
+.. versionadded:: 0.20.0
```

Comment:
0.20 -> 0.19

and I would also clarify what was added in 0.20.0 (only the ability to specify different dtypes for different columns with a dict)